docs/_index.md

```diff
@@ -7,7 +7,122 @@ type: "docs"
+          </div>
+        <div class="small-hover-card-div">
+            <h6>Texas Instruments TDA4VM</h6>
+        </div>
+      </a>
+</div>
+<div class="col hover-card hover-card-xxs">
+      <a href="installation/prepare/microcontrollers/">
+          <div class="hover-card-img">
+            <img src="installation/img/thumbnails/esp32-espressif.png" alt="E S P 32 - espressif" width="100%">
+          </div>
+        <div class="small-hover-card-div">
+            <h6>Espressif ESP32</h6>
+        </div>
+      </a>
+</div>
+<div class="col hover-card hover-card-xxs">
+      <a href="installation/prepare/jetson-agx-orin-setup/">
+          <div class="hover-card-img">
+            <img src="installation/img/jetson-agx-orin-setup/jetson-agx-orin-dev-kit.png" alt="Jetson A G X Orin Developer Kit" width="100%">
+          </div>
+        <div class="small-hover-card-div">
+            <h6>NVIDIA Jetson AGX Orin</h6>
+        </div>
+      </a>
+</div>
+<div class="col hover-card hover-card-xxs">
+      <a href="components/board/jetson/">
+          <div class="hover-card-img">
+            <img src="installation/img/thumbnails/jetson-xavier.jpeg" alt="Jetson Xavier NX Dev Kit" width="100%">
+          </div>
+        <div class="small-hover-card-div">
+            <h6>NVIDIA Jetson Xavier NX</h6>
+        </div>
+      </a>
+</div>
+<div class="col hover-card hover-card-xxs">
+      <a href="installation/prepare/rpi-setup">
+          <div class="hover-card-img">
+            <img src="installation/img/thumbnails/raspberry-pi-zero-2w.png" alt="Raspberry Pi" width="100%">
+          </div>
+        <div class="small-hover-card-div">
+            <h6>Raspberry Pi Zero 2W</h6>
+        </div>
+      </a>
+</div>
+<div class="col hover-card hover-card-xxs">
+    <a href="components/board/nanopi/">
+        <div class="hover-card-img">
+            <img src="installation/img/thumbnails/nanopi.jpeg" alt="FriendlyElec's NanoPi Mini Board" width="100%">
+        </div>
+        <div class="small-hover-card-div">
+            <h6>FriendlyElec NanoPi</h6>
+        </div>
+    </a>
+</div>
+<div class="col hover-card hover-card-xxs">
+    <a href="components/board/numato/">
+        <div class="hover-card-img">
+            <img src="installation/img/thumbnails/numato.png" alt="Numato GPIO Modules" width="100%">
+        </div>
+        <div class="small-hover-card-div">
+            <h6>Numato GPIO Modules</h6>
+        </div>
+    </a>
+</div>
+<div class="col hover-card hover-card-xxs">
+    <a href="components/board/numato/">
+        <div class="hover-card-img">
+            <img src="installation/img/thumbnails/pca9685.jpeg" alt="P C A 9685 Arduino I 2 C Interface" width="100%">
+        </div>
+        <div class="small-hover-card-div">
+            <h6>PCA9686 Arduino I<sup>2</sup>C Interface</h6>
+        </div>
+    </a>
+</div>
+</div>
+</div>
 
 {{< alert title="New to Viam?" color="tip" >}}
```

Comment:
I'd actually remove that and then change the try card to 

```
        <div class="landing-hover-card-padding yellow">
            <h4>Learn and Try</h4>
            <p style="text-align: left;">
            Learn about <a href="viam">the Viam platform in 3 minutes</a> and then
            <a href="try-viam">drive a Viam rover</a> from the comfort of your home or follow along with a <a href="tutorials"> tutorial</a>.</p>
            {{<gif webm_src="img/rover.webm" mp4_src="img/rover.mp4" alt="A Viam Rover moving about">}}
        </div>
```

and add another bullet on the configure card:

```
<li><a href="manage/fleet/robots/#control">Test your robot</a></li>
```